yogstation/code/game/gamemodes/gangs/gangs.dm

```diff
@@ -0,0 +1,59 @@
+//gang.dm
+//Gang War Game Mode
+GLOBAL_LIST_INIT(possible_gangs, subtypesof(/datum/team/gang))
+GLOBAL_LIST_EMPTY(gangs)
+/datum/game_mode/gang
+	name = "gang war"
+	config_tag = "gang"
+	antag_flag = ROLE_GANG
+	restricted_jobs = list("Security Officer", "Warden", "Detective", "AI", "Cyborg","Captain", "Head of Personnel", "Head of Security")
```

Comment:
Me and Altoids agreed that you should limit all the heads from starting as gang leaders. This still doesn't bar conversion of course, just bars them spawning as ones.